blue_level/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+    font-family: Arial, sans-serif;
+    background-color: rgb(238, 231, 213);
+    color: #3333cc;
+    text-align: center;
+  }
+  
+  
+  .main-container { 
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  #robotchef img {
```

Comment:
We shouldn't style by `id`, let's change it to `class`

Reply:
added class to HTML and updated CSS to reflect.